### ISSUE TITLE
[Bug] Crash with Elytra Slot

### ISSUE THREAD

**Issue by SaltSouls** [BeansGalaxy/Beans-Backpacks-2#121]
## Description:
So for some reason this mod is creating and registering the same curios capability as the elytra slot mod, causing the game to crash when using something like EMI to scroll through the lists of items. What I would like to suggest is that this mod only registers its capability if the elytra slot mod isn't loaded. That or make it so the elytra slot mod is a requirement if you plan on having basically the same functionality as it. This way it can avoid further issues/crashes when the two mods are used in combination with each other.

## Logs:
[latest.log](https://gist.github.com/SaltSouls/57414b7765d433552ef3284ca7a4c384) | [debug.log](https://gist.github.com/SaltSouls/b77070e107d398dd8ad83afed9f663aa)

**Comment by BeansGalaxy** [comment]
Long time no see!

Originally I had depended on Elytra Slot to wear it in the Back Slot but I ended up creating so many capabilities that mirrored theirs that I just moved the elytra over to use my code. I planned on just creating the rest of their capabilities in my own mod but never got around to it + feels kind of cheap to make their mod obsolete like that + their implementation is going to be much better than the after thought mine was gonna be.

I'll look at throwing out my implementation if Elytra Slot is installed and get our mods working together again.
